save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/progressbar/ReactCircleBuilder.kt

```diff
@@ -9,8 +9,11 @@ import react.react
  * @property hexColor color in hex format with leading `#` (example: #aabbcc)
  */
 enum class Color(val hexColor: String) {
+    DANGER("#dc3545"),
     GREEN("#00d500"),
```

Comment:
I would remove the old colors. They are still essentially the same.

Reply:
Decided to leave as it as I am not sure if AndreyK likes the changes.

Will remove though

Reply:
Removed your `enum class Color` and reused existing `enum class Colors`